iam_profile_faker/cli.py

```diff
@@ -38,30 +36,8 @@ def create_batch(count):
     click.echo(output)
 
 
-@click.command()
-@click.option('--count', type=int, default=100,
```

Comment:
I would suggest to keep this for now along with the API endpoints and drop only GraphQL. This can be useful to test some services by connecting to a user endpoint until we have the `dino-dev-preview` fully automated regarding profile population in ES.